Q7/BunnyEars.rb

```diff
@@ -0,0 +1,9 @@
+def bunny_ear(b)
+    if b == 0
```

Comment:
could be refactored as
return 0 if b == 0
return 2+...

Reply:
I refactored my code, thanks for the advice :)